homeassistant/components/webostv/media_player.py

```diff
@@ -30,8 +30,7 @@
     CONF_NAME,
     ENTITY_MATCH_ALL,
     STATE_OFF,
-    STATE_PAUSED,
-    STATE_PLAYING,
+    STATE_ON,
```

Comment:
Please remove the changes to the webostv integration.

Reply:
@MartinHjelmare I have not touch the file "homeassistant/components/webostv/media_player.py",I have compared this file.this file on iammeter branch is the same as dev branch.
should I update the 'media_player.py' in iammeter branch to add these two lines ?
thanks for your reply.

Reply:
You're first commit contains changes to these files. I'd start over in a new branch and PR based on latest dev branch.

Reply:
Hi,@MartinHjelmare,
what should I do to keep the pr process go forward at this step,should I just wait ?
or should I use some git tools to "remove" the commit history,sorry that I am not familiar with git command.

Reply:
If you're not familiar with git I suggest you start over in a fresh branch and submit a new PR. Create a new branch based on freshly pulled dev branch, apply your own changes, push and create a new PR.

Reply:
thanks for your advice,I will do that.